### PR TITLE
Use hoe's plugin rdoc

### PR DESCRIPTION
**Pull request by splattael**
Hi Ryan,

I've noticed that `hoe` provides its README and History as rdoc by using its own plugin `rdoc`.

There are more seattlerb projects which also use this plugin to make GitHub users very happy:
* https://github.com/seattlerb/minitest-bisect
* https://github.com/seattlerb/minitest-autotest
* https://github.com/seattlerb/minitest-sprint

This is awesome! :green_heart: 

After this commit `minitest`'s README and History read really nice on GitHub.

Note that I tried hard not to break gem build process by adjusting `Manifest.txt`.
Commands as `rake debug_gem` and `rake gem` still run smoothly after my change.

As I'm not very familiar with `hoe` it is very likely that I broke something. Please bare with me.
I happily receive feedback :smiley:

Kind regards,
  Peter

### CODE REVIEW

File: lib/minitest.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 require "minitest/parallel"
 
 ##
-# :include: README.txt
+# :include: README.rd
```
Comment (zenspider) [inline lib/minitest.rb]: You named the file `.rdoc`

Comment (splattael) [inline lib/minitest.rb]: @zenspider Thanks for the heads-up! Fixed and squashed.

**Comment by splattael** [comment]
@zenspider I've rebased this PR to the latest changes on `master`. It's now up-to-date again ;)

**Comment by gouravtiwari** [comment]
:+1: for readability

**Comment by splattael** [comment]
@zenspider Do you have any comments on this PR?
It's rebased to the latest master and ready to be merged :green_heart:

**Comment by zenspider** [comment]
This is done (and released)

**Comment by splattael** [comment]
Yes! :clap:

Thank you so much! :green_heart: :heart: :purple_heart:

**Pull request closed without merging**